content/pages/pricing.html

```diff
@@ -2,389 +2,263 @@
-              A question our users often have is what the difference is between Read the Docs Community and Read the Docs for Business. The features available on both platforms are the same. The primary difference is the audience and use cases that are supported.
-            </p>
-
-          </div>
-        </div>
-
-      </div>
-
-      <div class="ui vertical segment basic padded center aligned">
-        <a href="https://docs.readthedocs.io/en/stable/choosing-a-site.html#choosing-between-our-two-sites" class="ui button black huge">Choosing between our two sites</a>
-      </div>
-
+  </div>
+</section>
+
+{# Compare plan features with a table #}
+<section id="compare">
+  <div class="ui padded container">
+
+    {% call pricing.feature_table() %}
+      {{ pricing.feature_row("Team management", "Control organization access with team permissions", "fa-users", on_community=False) }}
+      {{ pricing.feature_row("Email support", "Get help with your projects and documentation", "fa-message-question", None, "2 business days", "2 business days", "1 business day") }}
+      {{ pricing.feature_row("API access", "Write your own integrations", "fa-sparkles") }}
+
+      {{ pricing.header_row("Authoring", "fa-typewriter") }}
+      {{ pricing.feature_row("Unlimited projects", "Manage and build any number of projects", "fa-infinity") }}
+      {{ pricing.feature_row("Public repositories", "Clone public repositories", "fa-globe") }}
+      {{ pricing.feature_row("Private repositories", "Clone private repositories", "fa-eye-slash", on_community=False) }}
+      {{ pricing.feature_row("Pull request preview", "Preview changes in review", "fa-code-compare") }}
+
+      {{ pricing.header_row("Documentation building", "fa-gears") }}
+      {{ pricing.feature_row("Automatic building", "Build every commit", "fa-gears") }}
+      {{ pricing.feature_row("Build concurrency", "Concurrent build processes", "fa-rectangle-vertical-history fa-swap-opacity", 2, 2, 4, 6) }}
+      {{ pricing.feature_row("Notifications", "Receive notifications build events", "fa-envelope-dot") }}
+      {{ pricing.feature_row("Environment variables", "Inject secrets into builds", "fa-lock-keyhole") }}
+      {{ pricing.feature_row("Automation rules", "Automatically manage versions", "fa-robot") }}
+
+      {{ pricing.header_row("Documentation hosting", "fa-browser") }}
+      {{ pricing.feature_row("Documentation versioning", "Host multiple versions of your project", "fa-layer-group") }}
+      {{ pricing.feature_row("Documentation localization", "Host your project localized in multiple locales", "fa-language") }}
+      {{ pricing.feature_row("Reader authentication", "Require access to view your documentation", "fa-shield-keyhole", on_community=False) }}
+      {{ pricing.feature_row("Single sign-on", "Sign on with using external provider", "fa-right-to-bracket", on_community=None, on_basic="GitHub and GitLab", on_advanced="GitHub and GitLab", on_pro="GitHub, GitLab, and Google") }}
```

Comment:
I feel like we're not selling the Pro plan enough here. I think it probably makes sense to split Google Auth out into its own line where only Pro has it.

Reply:
I also thought about highlighting these table cells differently, to provide more contrast. I can try that first, I felt that multiple rows defeated the purpose of building a table.

Reply:
Yeah, I'm leaning towards highlight:

![image](https://user-images.githubusercontent.com/1140183/201781548-f2fd5017-cbd9-482f-97a1-71fdc30e6f0d.png)

Reply:
Yea, that works well 👍